server/views/case-search.njk

```diff
@@ -50,75 +50,86 @@
     {%- from "components/search/macro.njk" import pacSearch -%}
     {{ pacSearch({
         search: search,
-        term: data.term,
-        courtCode: data.params.courtCode
+        term: term,
+        courtCode: data.params.courtCode,
+        searchError: searchError
     }) }}
 
-    <h3 class="govuk-heading-m">{{ data.totalElements if data.totalElements > 0 else 0}} search results for {{ data.term }}</h3>
-    {% set resultSize = data.items | length  %}
-    {% if data.items | length %}
-        {% block table %}
-            {%- from "govuk/components/table/macro.njk" import govukTable -%}
-                {{ govukTable(tableData) }}
-        {% endblock %}
-
-        {% set maxPagesDisplay = 5 %}
-        {% set startNum = currentPage - ((maxPagesDisplay - 1) / 2) %}
-        {% set endNum = currentPage + ((maxPagesDisplay - 1) / 2) %}
-        {% set totalPages = (data.totalElements / pageSize) | round(0, "ceil") %}
-
-        {% if startNum < 1 or totalPages <= maxPagesDisplay %}
-            {% set startNum = 1 %}
-            {% set endNum = maxPagesDisplay %}
-            {% elif endNum > totalPages %}
-            {% set startNum = totalPages - (maxPagesDisplay - 1) %}
+    {% if searchError | length == 0 %}
```

Comment:
Would there be a time when this was used for all errors? Make it a bit more generic? A question maybe for the wider team?

Reply:
Yea, I had this on mind. Can implement something very generic that works for all, However, I guess this is the only place where we have user input / error handling.